sdk/communication/azure-communication-jobrouter/src/main/java/com/azure/communication/jobrouter/implementation/models/BestWorkerModeInternal.java

```diff
@@ -3,26 +3,25 @@
 // Code generated by Microsoft (R) TypeSpec Code Generator.
 package com.azure.communication.jobrouter.implementation.models;
 
+import com.azure.communication.jobrouter.models.DistributionModeKind;
 import com.azure.communication.jobrouter.models.ScoringRuleOptions;
 import com.azure.core.annotation.Fluent;
 import com.azure.core.annotation.Generated;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeId;
 import com.fasterxml.jackson.annotation.JsonTypeInfo;
 import com.fasterxml.jackson.annotation.JsonTypeName;
 
 /**
  * Jobs are distributed to the worker with the strongest abilities available.
  */
-@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.PROPERTY, property = "kind")
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "kind", defaultImpl = BestWorkerModeInternal.class, visible = true)
 @JsonTypeName("bestWorker")
 @Fluent
 public final class BestWorkerModeInternal extends DistributionModeInternal {
 
     /*
-     * Define a scoring rule to use, when calculating a score to determine the best worker. If not set, will use a
-     * default scoring formula that uses the number of job labels that the worker labels match, as well as the number
-     * of label selectors the worker labels match and/or exceed using a logistic function
-     * (https://en.wikipedia.org/wiki/Logistic_function).
+     * Define a scoring rule to use, when calculating a score to determine the best worker. If not set, will use a default scoring formula that uses the number of job labels that the worker labels match, as well as the number of label selectors the worker labels match and/or exceed using a logistic function (https://en.wikipedia.org/wiki/Logistic_function).
```

Comment:
Seems Javadoc formatting broke?

Reply:
Does checkstyle no longer check char per line on Javadoc/comment?

I thought previously checkstyle would fail if line over 120 (exclude a few exceptions that line break not possible).

Reply:
I think the line wrap on Javadoc comment works on the new release 0.15.6.

Here is just comment, and line wrap not taking effect.